framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -4528,4 +4529,23 @@ public boolean containsSameByClass(
     protected WholeProgramInference createWholeProgramInference() {
         return new WholeProgramInferenceScenes(this);
     }
+
+    /**
+     * Does that {@link org.checkerframework.framework.qual.AnnotatedFor}, {@code anno} annotation
```

Comment:
I had to read this sentence a couple of times.  Can you remove "that `{@link ...}`", or change it to "Does an `{@link ...}` apply to this checker?"?